ui/src/components/Issuer/IssueAsset.tsx

```diff
@@ -70,6 +71,20 @@ const IssueAsset = () => {
                     </>}
                 <Radio toggle defaultChecked onClick={() => setIsPublic(!isPublic)}/>
             </div>
+
+            {!isPublic &&
+                <>
+                    <p>Add Observers</p>
+                    <p><i> Input each party Id seporated by a comma </i></p>
+                    <Form.Input
```

Comment:
We should use a `Form.TextArea` since the party Ids on DABL are long, and people might like to list them on separate lines

```
ledger-party-aaa-aaaa-aaaa,
ledger-party-bbb-bbbb-bbbb,
ledger-party-ccc-cccc-cccc,
...
```

Then after you `.split(',')`, you should also attach a `.map(p => p.trim())` just in case the newline character or extra spaces sneak in there as well